client/src/components/App.jsx

```diff
@@ -1,25 +1,46 @@
 const styyle = require('../style.scss');
+const axios = require('axios');
```

Comment:
Currently we have a mix of axios, jquery, and fetch. I think it'd be easy for me to switch to axios too since I've used it before but as I mentioned in Alex's PR review it's probably best to communicate beforehand and have all of us use the same library for ajax calls.